src/Pages/URLShortener/URLShortener.js

```diff
@@ -85,7 +85,10 @@ export default function URLShortenerPage(props) {
   async function maybeSubmitSearch() {
     console.debug('reached this call');
     const regex = /^[a-zA-Z0-9]+$/;
-    if (regex.test(searchQuery)) {
+    if (searchQuery === '') {
+      setInvalidSearch(false);
+      getCleezyUrls(undefined, searchQuery);
```

Comment:
```suggestion
      getCleezyUrls(page, searchQuery);
```
it should go back to the page they were on instead of undefined

Reply:
fixed